server.js

```diff
@@ -2,6 +2,75 @@ const express = require('express');
 
 const server = express();
 
-// your code here
+const Data = require('./data/accounts-model');
```

Comment:
Since data is a variable and its not a class or a component, maybe name it just with lowercase. This is just a suggestion.